ReactWindows/ReactNative.Shared.Tests/Internal/AssertEx.cs

```diff
@@ -1,4 +1,4 @@
-﻿using NUnit.Framework;
+using NUnit.Framework;
```

Comment:
no change here, maybe remove from PR?

Reply:
Yeah, not sure what happened, not even sure how to remove from PR :(

---
In reply to: [166077730](https://github.com/Microsoft/react-native-windows/pull/1615#discussion_r166077730) [](ancestors = 166077730)

Reply:
@reseul - one practice that I tend to follow is keep my number of commits in a PR small. If I only have 1 commit, for example, removing this file is as simple as resetting the one commit, unstaging / resetting the changes to this file, and re-commiting.

---
In reply to: [166089940](https://github.com/Microsoft/react-native-windows/pull/1615#discussion_r166089940) [](ancestors = 166089940,166077730)